integration_tests/integration/incidentDetailsEdit.cy.ts

```diff
@@ -272,10 +278,153 @@ context('Incident details (edit) - statement incomplete', () => {
       cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
       const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
       incidentDetailsPage.locationSelector().select('Workshop 2')
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="Yes"]').click()
       incidentDetailsPage.submitButton().click()
       cy.location().should(loc => {
         expect(loc.pathname).to.eq(adjudicationUrls.detailsOfOffence.urls.start(34))
       })
     })
+    it('should set DICOVERY button to Yes is DISCOVERY date same as INCIDENT date ', () => {
+      cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').should('be.checked')
+    })
+    it('should submit form successfully if DISCOVERY selected and all data entered and redirect to offence details page - change location - 2', () => {
+      cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.locationSelector().select('Workshop 2')
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').click()
+      incidentDetailsPage.timeInputHoursDiscovery().clear()
+      incidentDetailsPage.submitButton().click()
+      incidentDetailsPage
+        .errorSummary()
+        .find('li')
+        .then($errors => {
+          expect($errors.get(0).innerText).to.contain('Enter the time of the discovery')
+        })
+    })
+    it('should submit form successfully if DISCOVERY selected and all data entered and redirect to offence details page - change location - 1', () => {
+      cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.locationSelector().select('Workshop 2')
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').click()
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="Yes"]').click()
```

Comment:
I don't understand why you're selecting both 'yes' and 'no' radios here? You shouldn't need to select either if the date of discovery data is added to the stubs.
See comment about testing one thing per test - again you've used a test that was for testing a change in location input

Reply:
changes that cut and paster issue